test/images/volume/nfs/run_nfs.sh

```diff
@@ -22,21 +22,22 @@ function start()
     while getopts "G:" opt; do
         case ${opt} in
             G) gid=${OPTARG};;
-        esac
+            *) echo "accept: -G gid = ${OPTARG}"
+	esac
     done
-    shift $(($OPTIND - 1))
+    shift $((OPTIND - 1))
```

Comment:
Why did it change? What did it fix?
```
In ./test/images/volume/nfs/run_nfs.sh line 27:
    shift $(($OPTIND - 1))
             ^-----^ SC2004: $/${} is unnecessary on arithmetic variables
Ref: https://github.com/koalaman/shellcheck/wiki/SC2004
```
example:
**Problematic code**:
```
echo $(($n + ${arr[i]}))
```
**Correct code**: 
```
echo $((n + arr[i]))
```